### ISSUE TITLE
Npm install is installing v2 instead of v1

### ISSUE THREAD

**Issue by luanmuniz** [paypal/PayPal-node-SDK#346]
### Issue description

`npm install paypal-rest-sdk` is installing `v2.0.0-rc-1`

It seems that this was a confusion when deploying to npm. 
I'm saying this because v2 has no documentation, therefore, not production ready in my opinion.

**Comment by zaw-hlaing-bwar** [comment]
yeah I am also find the right document for v2!
paypal.configure is not a function in v2
is there any way to config in v2 or
Do I just need to use v1 instead?

**Comment by zaw-hlaing-bwar** [comment]
found a guide!!!!!
[doc](https://github.com/paypal/PayPal-node-SDK/tree/2.0-beta)

**Comment by luanmuniz** [comment]
@zaw-hlaing-bwar Unfortunately, the documentation for v2 is really poor. You will need to look at the code to know how to use most of it. I'm forcing the use of `v1.8.1` here. I don't know how ready is v2 yet.

**Comment by jaypatel512** [comment]
Hey @luanmuniz @zaw-hlaing-bwar !

That's a great catch. We just reverted `1.8.1` to latest and `2.0.0-rc-1` as beta.

However, do note that we are planning to make 2.x as latest once we have cleared out all known bugs, and improvements. So, if you are planning to use 1.x for a longer time, I would recommend using `~1` as your version number to make sure this does not happen to you when we make 2.x as latest. 

About the documentation for 2.x, is there anything you would want to see in our documentation that could help you migrate/integrate better ? We are open to suggestions. Leave a comment here, or create a new issue for those suggestions. 

I will close this issue as resolved.
